ValveKeyValue/ValveKeyValue/KVSerializerOptions.cs

```diff
@@ -26,6 +23,27 @@ public string[] Conditions
         /// </summary>
         public IIncludedFileLoader FileLoader { get; set; }
 
+        public KVSerializerOptions()
+        {
+            // TODO: In the future we will want to skip this for consoles and mobile devices?
+            Conditions.Add("WIN32");
```

Comment:
Is there a `WIN64` or any equivalent for 64-bit runtimes that we should be including here?

Reply:
Nope.